src/csync/csync_private.h

```diff
@@ -102,6 +102,8 @@ class ByteArrayRef
 
     friend bool operator==(const ByteArrayRef &a, const ByteArrayRef &b)
     { return a.size() == b.size() && qstrncmp(a.data(), b.data(), a.size()) == 0; }
+
+    QByteArray toByteArray() const { return QByteArray(data()); }
```

Comment:
:-1:  You need to pass the _size as well.

You don't use this, so just remove this